rebasehelper/patch_helper.py

```diff
@@ -279,6 +280,10 @@ def run_patch(cls, old_dir, new_dir, rest_sources, patches, **kwargs):
         cls.rest_sources = rest_sources
         cls.patches = patches
         cls.non_interactive = kwargs.get('non_interactive')
+        cls.favor_on_conflict = kwargs.get('favor_on_conflict')
+        if cls.favor_on_conflict:
+            # transform upstream to 'ours' and downstream to 'theirs' according to git strategy option
+            cls.favor_on_conflict = 'ours' if (cls.favor_on_conflict == 'upstream') else 'theirs'
```

Comment:
Not that it matters too much, but I think it would be less confusing to do this in `_git_rebase` using a local variable called `strategy_option` for instance.